lib/queue.rb

```diff
@@ -1,31 +1,32 @@
 class Queue
```

Comment:
You're implementing the Queue with a straight array, which works, but isn't the assignment and not the most efficient way.  The assignment is to use a circular buffer.